include/cppkafka/configuration.h

```diff
@@ -45,6 +45,8 @@
 
 namespace cppkafka {
 
+constexpr const char* library_name = "cppkafka";
```

Comment:
Can you just move this into `Consumer::~Consumer`? This is kind of out of place up here.